sleep/sleep.c

```diff
@@ -205,9 +207,9 @@ ENTRYPOINT(
 
             if (ConsoleMode) {
                 SetConsoleCursorPosition(GetStdHandle(STD_OUTPUT_HANDLE), ScreenInfo.dwCursorPosition);
-                YoriLibOutput(YORI_LIB_OUTPUT_STDOUT, _T("%02lli:%02lli  "), LongTimeToSleep / (1000 * 60), LongTimeToSleep / 1000);
+                YoriLibOutput(YORI_LIB_OUTPUT_STDOUT, _T("%02lli:%02lli:%02lli"), LongTimeToSleep / (1000 * 60 * 60), LongTimeToSleep / 1000 % 3600 / 60, LongTimeToSleep / 1000 % 3600 % 60);

```

Comment:
Isn't the "% 3600" in "% 3600 % 60" redundant?  Also, I know that the operators here evaluate correctly, but some brackets to make sure it's clear for the reader might be nice.